### DISCUSSION TITLE
Transfer image folder -> web page

### DISCUSSION THREAD

**Discussion by MathisHeriveau** [community/community#32577] [Other Feature Feedback, Questions, & Ideas]
Hey!
Today I tried to put the beginning of a website online.
So I put everything in a repositories and made settings to have a website.
Off, once on the site, we see that there are no images. And, by inspecting the source page, we can see that the images were not all transferred.

My site : https://mathisheriveau.github.io/BESST/
The repositories : https://github.com/MathisHeriveau/BESST

Thanks.

**Comment by airtower-luna** [comment]
It looks like you already found and fixed the absolute links that were causing the issue: https://github.com/MathisHeriveau/BESST/commit/43aa170540b3903138e408f75b55783cbfe568fc

I see it hasn't deployed yet, but I assume that'll just take a little while. :slightly_smiling_face:
